repository/kafka/docs/v0.2/configuration.md

```diff
@@ -0,0 +1,70 @@
+# Configuration 
+
+### Resources
+
+By default, KUDO Kafka resource configuration is set to the minimum recommended values for production usage. 
+But users can and should tune the configurations based on the workload requirements of their Kafka cluster.  
+
+##### Tuning the resources for the Kafka Cluster
+
+```
+kubectl kudo install kafka --instance=my-kafka-name \
+            -p ZOOKEEPER_URI=zk-zookeeper-0.zk-hs:2181,zk-zookeeper-1.zk-hs:2181,zk-zookeeper-2.zk-hs:2181 \
+            -p ZOOKEEPER_PATH=/custom-path \
+            -p BROKER_CPUS=3000m \
+            -p BROKER_COUNT=5 \
+            -p BROKER_MEM=4096m \
+            -p DISK_SIZE=20Gi \
+            -p MIN_INSYNC_REPLICAS=3 \
+            -p NUM_IO_THREADS=10 \
+            -p NUM_NETWORK_THREADS=5 
+          
+```
+
+##### Storage
+
+By default, the Kafka operator will use the default storage class of the Kubernetes cluster. 
+
+To deploy Kafka using a different storage class, you can use the parameter `STORAGE_CLASS`
+
+```
+kubectl kudo install kafka --instance=my-kafka-name -p STORAGE_CLASS=<STORAGE_CLASS_NAME>
+```
+
+##### Deploying an ephemeral Kafka cluster without persistence
+
+```
+kubectl kudo install kafka --instance=my-kafka-name -p PERSISTENT_STORAGE=false
+```
+
+Having `PERSISTENT_STORAGE` value `false` means that any data or logs inside the brokers will be lost after a pod restart or rescheduling.
+Deploying without persistent storage isn't recommended for production usage. 
+
+##### Metrics
+
+By default, the Kafka cluster will have the JMX Exporter enabled. You can check more information around how KUDO Kafka exposes metrics in [monitoring](./monitoring.md).
+
+##### Zookeeper PATH
+
+By default, the Kafka cluster will try to use a zookeeper path same as specified in `--instance` flag.
+You can get the instance name by `kubectl get instances` 
+
+To override the default zk path, use the `ZOOKEEPER_PATH` parameter. In next example the  `ZOOKEEPER_PATH` will be
+`/custom-path`
+
+```
+kubectl kudo install kafka --instance=my-kafka-name \
+  -p ZOOKEEPER_URI=zk-zk-0.zk-hs:2181,zk-zk-1.zk-hs:2181,zk-zk-2.zk-hs:2181 \
+  -p ZOOKEEPER_PATH=/custom-path
```

Comment:
this example does switch the zookeeper path.. but it also provides the zookeeper uri.   Is that necessary?

Reply:
please read the above sentence `To override the default zk path`

Reply:
I did read the entire section including the sentence above.   There is no clarity in the doc regarding the value / difference between URI and PATH.   you mention overriding the PATH... but they you override the URI and the PATH.

Reply:
I see this doc format can be confusing for some people. Let me rethink about that. And I will add docs in table format where all configurable parameters are listed. And some examples of how to change a parameter in a KUDO operator.